doc/nrf/glossary.rst

```diff
@@ -459,8 +459,8 @@ Glossary
       See :ref:`zephyr:west`.
 
    West manifest file
-      The file containing the metadata of the manifest repository.
-      In the |NCS| and Zephyr, it is :file:`west.yml`.
+      The main file describing the contents of a west workspace, which is located in the manifest repository.
+      In the |NCS| and Zephyr, it is named :file:`west.yml`.
```

Comment:
Rationale: I just think this is clearer. The file name is common to both cases, but it's not the exact same file.